### PR TITLE
Merge Banner Standard

### PR DESCRIPTION
**Pull request by alie-nel**
Not sure where to put the hoodie picture, can be moved. Could not find a way to add custom font and colours without changing the theme. AND I COULD NOT FIX THE TAB SIZE :(

### CODE REVIEW

File: src/components/Home/BannerStandard.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+interface BannerInfo {
```
Comment (e3lo) [inline src/components/Home/BannerStandard.tsx:1]: Add this component into routes so we can see its effect
Also you may want to pull from dev first so some of the styling changed can be updated

File: src/components/Home/BannerStandard.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            className="self-stretch md:self-auto text-center md:text-left text-4xl md:text-5xl font-bold">
+            {title}
+          </p>
+        </div>
+
+        <div id="hoodie-desc" className="md:w-3/4">
+          <p className="self-stretch md:self-auto text-center md:text-left text-base font-medium">{desc}</p>
+        </div>
+
+        <div
```
Comment (e3lo) [inline src/components/Home/BannerStandard.tsx:30]: Wrap with a <Link> from [react router](https://reactrouter.com/en/main/components/link) or replace div with link tag. Also add the link url to props

**Review by e3lo — COMMENTED** [review]
Just some small changes good work

File: src/components/Home/BannerStandard.tsx (hunk 2 above)
Comment (alie-nel) [reply src/components/Home/BannerStandard.tsx:30]: Did this but now the button's text has default colours :(

**Pull request merged**